src/features/wallet/helpers/calculateUMATotalValue.ts

```diff
@@ -0,0 +1,9 @@
+export default function calculateUMATotalValue(price: number, balance: string) {
```

Comment:
this is cool, but just fyi ethers does have this utility [commify](https://docs.ethers.io/v5/api/utils/display-logic/#utils-commify), although your implementation is nice, so this is just an fyi

Reply:
Good to know, figured ethers didn't have something like that.

Reply:
Any reason not to use that instead of the regex as a matter of simplicity?

Reply:
Nope! That works just fine. 

Fixed, with an improvement for the 0 balance case as well.